docs/config.json

```diff
@@ -644,7 +644,11 @@
             {
               "title": "Terraform Provider",
               "slug": "/management/dynamic-resources/terraform-provider/"
-            }
+            },
+	    {
+	      "title": "Import Teleport Resources into Terraform",
+	      "slug": "/management/dynamic-resources/import-terraform-resources/"
```

Comment:
I like the distinct title of this article, but I'm left puzzled over _Terraform Provider_ in contrast. Might be worth clarifying the goal of that other guide too?